api/src/main/java/org/openmrs/module/insuranceclaims/api/model/AbstractBaseOpenmrsData.java

```diff
@@ -0,0 +1,30 @@
+package org.openmrs.module.insuranceclaims.api.model;
+
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+import org.openmrs.BaseOpenmrsData;
+
+public abstract class AbstractBaseOpenmrsData extends BaseOpenmrsData {
+
+	private static final String ID_FIELD_NAME = "id";
+
+	private static final long serialVersionUID = -1781076310482589522L;
```

Comment:
serialVersionUID is redundant because this class is abstract

Reply:
changed